niceman/interface/tests/test_execute.py

```diff
@@ -122,8 +122,8 @@ def test_trace_docker(docker_container, trace_info):
 # TypeError under Python 2.
 @pytest.mark.skipif(os.environ.get('NICEMAN_TESTS_NONETWORK'),
                     reason="No network")
-@pytest.mark.skipif("cmd:apt-cache" in external_versions,
-                    reason="Not apt-cache")
+@pytest.mark.skipif("cmd:apt-cache" not in external_versions,
+                    reason="No apt-cache")
```

Comment:
good catch! 
although functionally identical ATM (see below) I wonder if more correct form would be `not external_versions["cmd:apt-cache"]`?

```shell
$ python -c 'from niceman.support.external_versions import external_versions as ev; print("cmd:apt-cache" in ev)'                                 

2018-12-10 10:11:24,024 [ERROR  ] Failed to start ['apt-cache', '-v']'': [Errno 2] No such file or directory [subprocess.py:_execute_child:1025] 
False
(niceman-dev) [yhalchen@discovery7 niceman]$ 
$ python -c 'from niceman.support.external_versions import external_versions as ev; print(ev["cmd:apt-cache"])'                                      
2018-12-10 10:11:43,191 [ERROR  ] Failed to start ['apt-cache', '-v']'': [Errno 2] No such file or directory [subprocess.py:_execute_child:1025] 
None
```

Reply:
Sorry, I'm not following why the `__getitem__` variant would be more correct.

Reply:
I thought that it would make sense to eventually return `None` if nothing is installed, while `__contains__` to return True if we already "know" the version (which could be None).  But now I remembered that you fixed in DataLad recently and brought here the:
```shell
--- a/datalad/support/external_versions.py
+++ b/datalad/support/external_versions.py
@@ -225,7 +225,7 @@ class ExternalVersions(object):
         return self._versions.keys()
 
     def __contains__(self, item):
-        return item in self._versions
+        return bool(self[item])
```
so I guess we will stick to this semantic, sorry for the noise